### PR TITLE
feat(edpa): throttle VID worker RPCs

### PR DESCRIPTION
**Pull request by Marco-Premier**
Throttle metadata and Secure Computation RPCs across memoized and non-memoized VID Labeling TEE workers.

Issue: #4427

### CODE REVIEW

**Comment by wfa-reviewable** [comment]
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4440)

File: src/main/kotlin/org/wfanet/measurement/edpaggregator/BaseVidLabelingTeeAppRunner.kt [hunk 1]
```diff
@@ -38,6 +40,43 @@ abstract class BaseVidLabelingTeeAppRunner(
   private val hadoopConfigurationFor: (StorageConfig) -> Configuration
 ) : BaseTeeAppRunner() {
 
+  @CommandLine.Option(
+    names = ["--kingdom-rpc-min-interval"],
+    description = ["Minimum interval between outbound VID Repository RPCs."],
+    defaultValue = "500ms",
+  )
+  private lateinit var kingdomRpcMinInterval: Duration
```
Comment (stevenwarejones) [inline src/main/kotlin/org/wfanet/measurement/edpaggregator/BaseVidLabelingTeeAppRunner.kt:49]: Please make these four options use strict duration parsing and add CLI parsing tests for negative and partially malformed values. commandLineMain registers the common-jvm human-readable Duration converter, which delegates to the same substring-based String.toDuration() called out in #4439: --metadata-read-rpc-min-interval=-1s becomes positive 1s, and 500msjunk becomes 500ms, so fromMinimumIntervals() cannot reject them. A strict Picocli converter, ideally shared with the environment parser, should validate the entire value before conversion.

Comment (Marco-Premier) [reply src/main/kotlin/org/wfanet/measurement/edpaggregator/BaseVidLabelingTeeAppRunner.kt:49]: Done.

**Review by Marco-Premier — COMMENTED** [review]
> @Marco-Premier made 1 comment.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4440)* status: 0 of 40 files reviewed, 1 unresolved discussion (waiting on stevenwarejones).

File: src/main/kotlin/org/wfanet/measurement/edpaggregator/BaseVidLabelingTeeAppRunner.kt (hunk 1 above)
Comment (stevenwarejones) [reply src/main/kotlin/org/wfanet/measurement/edpaggregator/BaseVidLabelingTeeAppRunner.kt:49]: Verified — all four worker options now use the shared strict Picocli converter, and the shared parser rejects negative and partially malformed values before conversion. The CLI regression tests and all affected worker suites pass.

**Review by stevenwarejones — CHANGES_REQUESTED** [review]
> @stevenwarejones+AGNT made 1 comment.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4440)* status: 0 of 40 files reviewed, 1 unresolved discussion (waiting on stevenwarejones).

File: src/main/kotlin/org/wfanet/measurement/edpaggregator/BaseVidLabelingTeeAppRunner.kt (hunk 1 above)
Comment (Marco-Premier) [reply src/main/kotlin/org/wfanet/measurement/edpaggregator/BaseVidLabelingTeeAppRunner.kt:49]: Done.

**Review by Marco-Premier — COMMENTED** [review]
> @Marco-Premier made 1 comment.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4440)* status: 0 of 40 files reviewed, 1 unresolved discussion (waiting on stevenwarejones and stevenwarejones+AGNT).

**Review by stevenwarejones — COMMENTED** [review]
> @stevenwarejones reviewed 40 files and all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4440)* status: all files reviewed, 1 unresolved discussion (waiting on stevenwarejones).

**Review by stevenwarejones — APPROVED** [review]
> @stevenwarejones resolved 1 discussion.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4440)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on Marco-Premier).

**Review by laureanobrs — APPROVED** [review]
> @laureanobrs reviewed 40 files and all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4440)* status: all files reviewed, 2 unresolved discussions (waiting on Marco-Premier).

**Review by Marco-Premier — COMMENTED** [review]
> @Marco-Premier partially reviewed 41 files and resolved 2 discussions.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4440)* status: 38 of 41 files reviewed, all discussions resolved (waiting on laureanobrs, stevenwarejones, and stevenwarejones+AGNT).

**Review by Marco-Premier — COMMENTED** [review]
> @Marco-Premier reviewed 5 files and all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4440)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on laureanobrs and stevenwarejones).
